lib/action_cable/connection/identification.rb

```diff
@@ -27,7 +27,7 @@ def connection_identifier
 
       private
         def connection_gid(ids)
-          ids.map { |o| o.to_global_id.to_s }.sort.join(":")
+          ids.map { |o| (o.try(:to_global_id) || o.try(:to_s)).to_s }.sort.join(":")
```

Comment:
What I mean by undone is that `nil.try(:to_s)` returns `nil` and then `nil.to_s` returns an empty string.